src/services/historique-cras/historique-cras.class.js

```diff
@@ -0,0 +1,75 @@
+const { NotFound } = require('@feathersjs/errors');
+const { Service } = require('feathers-mongodb');
+const { ObjectId } = require('mongodb');
+const dayjs = require('dayjs');
+
+const { userAuthenticationRepository } = require('../../common/repositories/user-authentication.repository');
+const { userIdFromRequestJwt } = require('../../common/utils/feathers.utils');
+const statsCras = require('../stats/cras');
+
+exports.HistoriqueCras = class HistoriqueCras extends Service {
+  constructor(options, app) {
+    super(options);
+
+    app.get('mongoClient').then(db => {
+      this.Model = db.collection('cras');
+    });
+
+    app.get('/historique-cras/liste', async (req, res) => {
+      const db = await app.get('mongoClient');
+      const user = await userAuthenticationRepository(db)(userIdFromRequestJwt(req));
+      const maxDate = new Date(dayjs().subtract(1, 'month'));
+      const theme = req.query.theme !== 'null' ? req.query.theme : null;
+      let query = {
+        'conseiller.$id': new ObjectId(user.entity.oid),
+        'createdAt': {
+          '$gte': maxDate
+        }
+      };
+      if (theme) {
+        query = {
+          'cra.themes': {
+            '$in': [theme]
+          },
+          'conseiller.$id': new ObjectId(user.entity.oid),
+          'createdAt': {
+            '$gte': maxDate
+          }
```

Comment:
ok tu peux simplifier ici avec le spread en ajoutant au query déjà construit uniquement la partie { 'cra.themes': { '$in': [theme] }